### ISSUE TITLE
Midpoint Review - IA Feedback - Debt Resolution, Streamlined Waiver

### ISSUE THREAD

**Issue by erinrwhite** [department-of-veterans-affairs/va.gov-team#63343]
## VFS actions
- [x] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [x] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [x] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [63393](https://app.zenhub.com/workspaces/vsa---debt-607736a6c8b7e2001084e3ab/issues/gh/department-of-veterans-affairs/va.gov-team/63393) - [SW] Midpoint Review - IA links and typo updates (63343) **Closed**
  - [x] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [63395](https://app.zenhub.com/workspaces/vsa---debt-607736a6c8b7e2001084e3ab/issues/gh/department-of-veterans-affairs/va.gov-team/63395) - [SW] Midpoint Review - IA clarification updates (63343) (Post Launch)
- [x] Close the ticket when all feedback has been addressed.

## Thoughts/questions

- I'm excited that this is in motion and will relieve debt for so many Veterans! 
- Small thing that may or may not come up during testing - the tab order on the form fields in UXpin was reversed for me - I'd hit tab and it would go back a field rather than forward. Not sure if it was just my browser, and not sure if this is even something that'll be an issue during testing, but wanted to mention it.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Must:** Include an exit route or next step for users who see error states for overstatements or copays [Example error page](https://preview.uxpin.com/6940f648e5994b8fad9550934a5530b4cd73cacb#/pages/163500322/simulate/sitemap?mode=cvhid) - current error pages are a dead end
- **Must:** Privacy policy link next to the checkbox on the [final review page](https://preview.uxpin.com/bee98dfe15974e66ea5958c976cb43a2c08ecabc#/pages/163781815/simulate/sitemap?mode=cvhid) should have an indicator that the link opens in a new window so that users know they can review the link while staying in the form flow.
- **Must:** Fix small typos, apologies if this is already in the hopper or on another ticket:
  - [Your spouse’s info](https://preview.uxpin.com/bee98dfe15974e66ea5958c976cb43a2c08ecabc#/pages/163781759/simulate/sitemap?mode=cvhid) - “mariatal” should be marital
  - [Your current job deductions](https://preview.uxpin.com/bee98dfe15974e66ea5958c976cb43a2c08ecabc#/pages/163781782/simulate/sitemap?mode=cvhid) and [Spouse deduction amounts](https://preview.uxpin.com/bee98dfe15974e66ea5958c976cb43a2c08ecabc#/pages/163864120/simulate/sitemap?mode=cvhid) - "Retirment" should be Retirement
- **Consider:** which step the [You can skip questions](https://preview.uxpin.com/bee98dfe15974e66ea5958c976cb43a2c08ecabc#/pages/163781813/simulate/sitemap?mode=cvhid) page H2 should display. It's currently listed under Step 7: Review Information, which is a bit of a leap after users are on step 2 or 3.
- **Consider:** [Final review page](https://preview.uxpin.com/bee98dfe15974e66ea5958c976cb43a2c08ecabc#/pages/163781815/simulate/sitemap?mode=cvhid) - Statement of truth mentions bankruptcy history, but it wasn’t asked about in the UXpin user flow. Might cause confusion for Veterans if they see this and wonder why they weren't asked about it.
- **Consider:** Continue working with CAIA on [confirmation page](https://preview.uxpin.com/bee98dfe15974e66ea5958c976cb43a2c08ecabc#/pages/163781844/simulate/sitemap?mode=cvhid) content for scannability and clarity to help the Veteran quickly understand that their information has been submitted and what they need to do next. "You don't need to do anything else at this time" might be the most important sentence on the page, and it's down the page a bit.

**Comment by HeatherRienks** [comment]
FYI @Tom-Davis

**Comment by mgleo** [comment]
Hi @erinrwhite, thank you for this feedback. Can you provide more information for language and where we should link users who land on an error state page? Thanks!

**Comment by erinrwhite** [comment]
@mgleo My first instinct is a "Go back to VA.gov" action link below the alert message as a final signal that folks cannot proceed further in the flow. This aligns with our practice of including a "Go back to VA.gov" link at the very end of forms. If y'all have strong opinions or evidence for a more helpful action link, though, please go with that.

**Comment by mgleo** [comment]
@erinrwhite Thank you. That makes sense, I've added the "Go back to VA.gov" action link to the [prototype](https://preview.uxpin.com/6940f648e5994b8fad9550934a5530b4cd73cacb#/pages/163500322?mode=cvhid) and we'll add it on production as well.

Updated the [Privacy Policy section on the prototype](https://preview.uxpin.com/6940f648e5994b8fad9550934a5530b4cd73cacb#/pages/163500446/simulate/sitemap?mode=cvhid) and that will be updated on production. 

I also corrected those typos on the prototype, they were only present on the prototype and not on staging/production.
